setuptools_rust/build.py

```diff
@@ -505,6 +505,9 @@ def _cargo_args(
         if ext.args is not None:
             args.extend(ext.args)
 
+        if "_SETUPTOOLSRUST_BUILD_STD" in os.environ:
```

Comment:
I guess we learned we don't need this and can just have `CARGO_UNSTABLE_BUILD_STD=true`.